src/server/lib/middleware/csrf.ts

```diff
@@ -0,0 +1,49 @@
+import csrf from 'csurf';
+import { NextFunction, Request } from 'express';
+import { ResponseWithLocals } from '@/server/models/Express';
+import { getConfiguration } from '@/server/lib/configuration';
+
+const isHttps = getConfiguration().isHttps;
```

Comment:
Prefer destructuring:

```suggestion
const { isHttps } = getConfiguration();
```